configs/kwinrc

```diff
@@ -2,5 +2,4 @@
 update_info=kwin.upd:replace-scalein-with-scale,kwin.upd:port-minimizeanimation-effect-to-js,kwin.upd:port-scale-effect-to-js,kwin.upd:port-dimscreen-effect-to-js,kwin.upd:auto-bordersize,kwin.upd:animation-speed,kwin.upd:desktop-grid-click-behavior,kwin.upd:no-swap-encourage,kwin.upd:make-translucency-effect-disabled-by-default,kwin.upd:remove-flip-switch-effect,kwin.upd:remove-cover-switch-effect,kwin.upd:remove-cubeslide-effect,kwin.upd:remove-xrender-backend,kwin.upd:enable-scale-effect-by-default
 
 [Compositing]
-Enabled=false
```

Comment:
Why you want composition enabled by default?

Reply:
Without the composer, the tigervnc client-server connection is unstable. It causes the client to close

Reply:
I get it, but on pi4 it may make the whole thing clunky

Reply:
I didn't notice any problem on my pi 4. I realized these were changes on pi 4 before testing on pi 5
Did you test on your pi 4 with compositor then without?

Reply:
yes, I sometimes turn it on (also for some app like collimator circle is needed) and I noticed on the pi it is clunky after a while, haven't noticed any big difference on the pi5 tho. Maybe we could generate this file on the first boot checking which model we run on?

Reply:
Did you test with the vc4-kms-v3d driver and file modifications or with the current configuration?